app/assets/stylesheets/layout/navbar.css.scss

```diff
@@ -0,0 +1,87 @@
+nav { 
+    color: white;
+    background: $primary-color-dark;
+    height: 50px;
+}
+
+.nav-dropdown-image {
+    display: inline-block;
+    position: relative;
+    top: 7px;
+    left: 4px;
+}
+
+.nav-selected { // this highlights the nav element linking to the current page
+    font-weight: bold;
+}
+
+.nav-li { // this describes a elements or li elements  or .nav-submenus
```

Comment:
`<a>` and `,<li>` makes a little more sense. I almost gave a grammatical correction for "a elements"